generators/entity-server/templates/src/main/java/package/service/impl/EntityServiceImpl.java.ejs

```diff
@@ -78,14 +78,14 @@ import static org.elasticsearch.index.query.QueryBuilders.*;
 <%_ } _%>
 
 /**
- * Service Implementation for managing <%= entityClass %>.
+ * Service Implementation for managing <%= asEntity(entityClass) %>.
  */
 @Service<% if (databaseType === 'sql') { %>
 @Transactional<% } %>
 public class <%= serviceClassName %><% if (service === 'serviceImpl') { %> implements <%= entityClass %>Service<% } %> {
 
     private final Logger log = LoggerFactory.getLogger(<%= serviceClassName %>.class);
-<%- include('../../common/inject_template', {viaService: viaService, constructorName: serviceClassName, queryService: false}); -%>
+<%- include('../../common/inject_template', {asEntity: asEntity, asDto: asDto, viaService: viaService, constructorName: serviceClassName, queryService: false}); -%>
```

Comment:
since we are on node 8+ can you check if the shortHand syntax works here?
```suggestion
<%- include('../../common/inject_template', {asEntity, asDto, viaService, constructorName: serviceClassName, queryService: false}); -%>
```

Reply:
OK this seems to work well. I have made the change everywhere.